icon_validator/exceptions.py

```diff
@@ -2,6 +2,7 @@
 NO_CON_VERSION_CHANGE = "Connection details have changed but no version bump has occurred."
 INVALID_CON_VERSION_CHANGE = "Connection details have not changed but the connection version was bumped."
 INCORRECT_CON_VERSION_CHANGE = "Connection version has been bumped incorrectly."
+FIRST_TIME_CONNECTION_ISSUE = "First time connection version is supplied does not match expected value."
```

Comment:
Can we update this to mention version please? It could just be FIRST_TIME_CON_VERSION_ISSUE to line up with others?

Reply:
Updated now :)